rclcpp/include/rclcpp/node_interfaces/node_logging.hpp

```diff
@@ -49,13 +53,22 @@ class NodeLogging : public NodeLoggingInterface
   const char *
   get_logger_name() const override;
 
+  RCLCPP_LOCAL
+  void
+  add_logger_service(
+    rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_base,
+    rclcpp::node_interfaces::NodeServicesInterface::SharedPtr node_services);
+
```

Comment:
Can we put `create the logger service` in `NodeBuiltinExecutorImpl::add_logger_service`?

Reply:
Okay